### DISCUSSION TITLE
Discrepancy in Volume Flow Rate Measurements

### DISCUSSION THREAD

**Discussion by NhatNguyenEBC** [firemodels/fds#13515] [Q&A] [answered]
Hello everyone,

I recreated my problem on a simple cube simulation using Fire Dynamics Simulator (FDS). I have encountered an issue with the volume flow rate measurements. Here is the setup of my simulation:

- Geometry: A simple cube.
- Inlets: Two inlets at ground level with "open" as the boundary condition.
- Outlets: Two outlets on the ceiling with a set volume flow rate. Basically modelling mechanical exhaust vents.

To measure the volume flow rate, I have used two different types of devices:

- A device that directly measures the quantity "volume flow". (I have also tried "volume flow wall" as suggested in older discussions).
- A device that measures "normal velocity" and integrates over the corresponding surface to calculate the volume flow rate.

The problem I am facing is that these two measurement methods yield different results for one inlet but match for the other inlet/outlets.

Could anyone help me understand why this discrepancy might be occurring?

Thank you!
[Example_case.txt](https://github.com/user-attachments/files/17233912/Example_case.txt)

**Comment by drjfloyd** [comment, marked as answer]
NORMAL VELOCITY is a wall quantity. It measures the velocity at the face of the wall and has no impact from what is happening in the gas cells adjacent to the wall.  

VOLUME FLOW is a gas phase quantity. Velcoities in FDS are stored at grid cell boundaries. When you give an XB for measuring volume flow, FDS snaps the plane to the closet grid cell boundary and gets the velocity at that face. I suspect that what is happening is that in most cases the XB is snapping to the solid face. When this happens the gas phase velocity at the solid face is going to be the same as the value stored for the wall cell. The face that doesn't match is likely snapping to the gas phase face of the wall adjacent cell. The flow in or out of the wall is going to entrain air and some of that air is going to come in from the sides of the gas cells around the perimter of the vent. That volume flow is either going to add to or remove from the volume flow at the face of the gas cells next to the wall.

![image](https://github.com/user-attachments/assets/39a426bd-8dd0-4957-8854-3d7319ad628f)
